app.rb

```diff
@@ -0,0 +1,194 @@
+require './student'
+require './teacher'
+require './book'
+require './rental'
+
+class App
+  def initialize
+    # Print commands
+    @command = {
+      '1': 'List all books',
+      '2': 'List all people',
+      '3': 'Create a person',
+      '4': 'Create a book',
+      '5': 'Create a rental',
+      '6': 'List all rentals for a given person id',
+      '7': 'Exit'
+    }
+    # List all books.
+    @books = []
+    # List all people.
+    @peoples = []
+  end
+
+  # display commands
+  def display_command
+    @command.each do |index, command|
+      puts "#{index} -#{command}"
+    end
+  end
+
```

Comment:
[REQUIRED] I get an error when I try to review your file. This is preventing me from carrying out a proper review.

<img width="995" alt="Screenshot 2023-01-05 at 18 44 26" src="https://user-images.githubusercontent.com/62938735/210847126-e50bd69e-32d5-4089-a9d3-b6db6a9b4f75.png">